### PR TITLE
Form info fixes

### PR DESCRIPTION
**Pull request by ejulcsi**
#### Is connection necessary to test? If so which network?
- [ ] local RPC
- [ ] Rinkeby
- [ ] Main Ethereum Network

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Building with commit 45e48c9e3ae59e2830aa85338aa849bea6cb95d3

https://app.netlify.com/sites/augmint/deploys/5cd048ecd53476000787812f

File: src/containers/loan/newLoan/LoanProductSelector.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ class LoanProductSelector extends React.Component {
                     </Pgrid.Column>
                     <Pgrid.Column className="loan column right" size={{ tablet: 1, desktop: 2 / 5 }}>
                         <Message info style={{ margin: 0 }}>
+                            <p>
+                                <strong>60% loan-to-value (LTV)</strong>
```
Comment (phraktle) [inline src/containers/loan/newLoan/LoanProductSelector.js]: Let's not hardcode this, instead take from loan product info.

File: src/containers/lock/containers/LockForm/index.js [hunk 2]
```diff
@@ -283,6 +283,11 @@ class LockContainer extends React.Component {
                             id="selectedLockProduct"
                             name="lockTerms"
                             data-testid="lock-product-selector"
+                            info={
+                                this.state.selectedProduct
+                                    ? `A-EUR unlock only available after ${this.state.selectedProduct.durationText}`
```
Comment (phraktle) [inline src/containers/lock/containers/LockForm/index.js]: make this a more scary red warning, after "on ... you get back...".

**Comment by phraktle** [comment]
Closing, features moved to separate PRs (#612, #613, #614, #615 , #616).

**Pull request closed without merging**